CONTRIBUTING.md

```diff
@@ -33,7 +33,7 @@ outlined on that page and do not file a public issue.
 
 Same as in [README](README.md) with the exception of:
 ```
-python setup.py develop
+pip install -e ".[dev]"
```

Comment:
We may also want to change the instructions in [README.md](https://github.com/facebookresearch/multimodal/blob/main/README.md#building-from-source)